hack/tools/create_kind.sh

```diff
@@ -54,6 +65,7 @@ install_certs() {
 certs_delete() {
     if [[ "$OSTYPE" == "darwin"* ]]; then
       security remove-trusted-cert  ../registry/ca.crt
+      security delete-certificate -c themeshfordata
     fi
     rm -rf ../registry
```

Comment:
Not related to this PR but don't we need to clean it up from the global CA folder as well?

Reply:
I'm not sure what you mean with the global CA folder. Can you please elaborate? 
The default is to store the certificate in the user cert store. And this removes it from the default user cert store. (Adding it in the admin cert store would require the `-d` flag when adding the trusted cert.) What I noticed is that for trusting/untrusting a certificate it's in need for the user password.

Reply:
My question is where do you remove the registry certificate from the user cert store in ubuntu (/usr/local/share/ca-certificates/)?. Also do you think we should create a file with a more unique name than ca.crt? Also after removal a call to update-ca-certificates with the --fresh (or -f) flag is needed.

Reply:
That's a good question. Currently it's not removed from the ubuntu ca-certificates. I did not test this on a ubuntu machine for some time. I'll check it out.